geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyJndiBindingCommand.java

```diff
@@ -65,8 +70,9 @@ public Result destroyJDNIBinding(
       Element existingBinding =
           service.getXmlElement("cluster", "jndi-binding", "jndi-name", jndiName);
       if (existingBinding == null) {
-        return ResultBuilder.createUserErrorResult(
-            CliStrings.format("Jndi binding with jndi-name \"{0}\" does not exist.", jndiName));
+        throw new EntityNotFoundException(
```

Comment:
These exceptions get caught and then wrapped into an `ErrorResult` by the `OperationInvoker`, right?  We're not actually changing behavior as it would effect, say, a `gfsh` script?

Reply:
Yes. the exceptions would be caught and translated into `Error or Success` result based on `ifExists`.